.github/workflows/array-api.yml

```diff
@@ -1,5 +1,9 @@
 name: Array API coverage tests
-on: [push]
+
+on:
+  push: # Run on all branches
+  schedule:
+    - cron: "0 8 * * *" # Runs daily at 00:00 UTC
```

Comment:
```suggestion
    - cron: "0 8 * * 1" # Runs weekly at 08:00 UTC on Monday
```

Reply:
I think a weekly run is sufficient. If a periodic test run like this fails, it would be great if a GH issue could be opened (see https://github.com/Quantco/glum/blob/main/.github/workflows/daily.yml#L48-L69 for an example).